Source/Core/Prune/DependencyEvaluator.cs

```diff
@@ -25,7 +25,7 @@ public class DependencyEvaluator : ReadOnlyVisitor
     public List<Declaration[]> incomingSets = new();
     public HashSet<Type> types = new();
     
-    private static bool ExcludeDep(Declaration declaration)
+    public static bool ExcludeDep(Declaration declaration)
```

Comment:
What does it fix that private became public here? I suggest you add a comment to the function about why it needs to be public of it is not testable (e.g. for APIs), in case someone later wants to revert it.

Reply:
Ok, if you don't intend to merge this PR first since you are including it in #450, which makesuse of ExcludeDep, you can ignore my comment.

Reply:
Pretty sure it can be `private` in #450 as well.